src/main/java/de/zalando/aruha/nakadi/controller/CursorsController.java

```diff
@@ -37,18 +39,30 @@ public CursorsController(final CursorsCommitService cursorsCommitService,
         this.featureToggleService = featureToggleService;
     }
 
+    @RequestMapping(path = "/subscriptions/{subscriptionId}/cursors", method = RequestMethod.GET)
+    public ResponseEntity<?> getCursors(@PathVariable("subscriptionId") final String subscriptionId, final NativeWebRequest request) {
+        if (!featureToggleService.isFeatureEnabled(HIGH_LEVEL_API)) {
+            return new ResponseEntity<>(NOT_IMPLEMENTED);
+        }
+
+        try {
+            return status(OK).body(cursorsCommitService.getSubscriptionCursors(subscriptionId));
```

Comment:
I think that assigning the result from `cursorsCommitService.getSubscriptionCursors(subscriptionId)` to a temp variable would improve clarity, since it would make it explicit what the return type is.